src/sourceInfo.ts

```diff
@@ -0,0 +1,88 @@
+import { google } from '../build/pbjs';
+import FileDescriptorProto = google.protobuf.FileDescriptorProto;
+
+export type FieldID = number;
+
+export const Fields = {
+    file: {
```

Comment:
I don't have it setup to auto-run, but there is a `.prettierrc` checked in; can you run prettier on this?

Reply:
@stephenh Done.